### ISSUE TITLE
Query error

### ISSUE THREAD

**Issue by Anfoni** [rpi-dotcio/phpLicenseWatcher#110]
**Describe the bug**
license_util.php is not returning proper polling data.

**To Reproduce**
1. Go to 'installation directory root /tools/phpLicenseWatcher-master/'
2. run _php licnese_util.php_

[root@adminMachine]$ php license_util.php
PHP Fatal error:  Uncaught Error: Call to a member function bind_param() on bool in /tools/eda/phplicensewatcher/phpLicenseWatcher-master/license_util.php:58
Stack trace:
#0 /tools/eda/phplicensewatcher/phpLicenseWatcher-master/license_util.php(8): update_servers(Object(mysqli), Array)
#1 {main}
  thrown in /tools/eda/phplicensewatcher/phpLicenseWatcher-master/license_util.php on line 58

**Expected behavior**
Output of polling data to display on php website.

**Screenshots**
![image](https://user-images.githubusercontent.com/45222272/152479284-1a3aec22-0399-4524-94df-8fd85cc3d8e4.png)

**Desktop (please complete the following information):**
 - OS: centos7
 - Browser firefox

**Additional context**

**Comment by pbailie** [comment]
So far I am unable to recreate the bug.

But the error message would be caused by a problem with either database communication or an error issued by the database.  Could you look at _check installation_ under _admin_?  There is a check for database connectivity.

**Comment by Anfoni** [comment]
Everything looks good:

![image](https://user-images.githubusercontent.com/45222272/152616058-5c95ee5d-ebda-4548-b809-79e004679fcc.png)


In the config.php:
_$db_hostname="localhost";
$db_username="lic_user";
$db_password="removed for security";
$db_database="licenses";_

**Comment by Anfoni** [comment]
When reading in the table, I used the one found here: phpLicenseWatcher-master/database/phplicensewatcher.sql

**Comment by Anfoni** [comment]
Sorry, I figured it out, I had to grant the lic_user all privilege's on that database and then the script worked.


_[anthony@usw1-cad phpLicenseWatcher-master]$ mysql -u root -p
Enter password: 
Welcome to the MySQL monitor.  Commands end with ; or \g.
Your MySQL connection id is 126
Server version: 5.7.37 MySQL Community Server (GPL)

Copyright (c) 2000, 2022, Oracle and/or its affiliates.

Oracle is a registered trademark of Oracle Corporation and/or its
affiliates. Other names may be trademarks of their respective
owners.

Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.

mysql> grant all privileges on licenses.* to 'lic_user'@'%';
Query OK, 0 rows affected (0.00 sec)_
